vkubelet/pod.go

```diff
@@ -105,7 +105,21 @@ func (s *Server) startPodSynchronizer(ctx context.Context, id int) {
 	logger.Debug("Start pod synchronizer")
 
 	for event := range s.podCh {
-		s.syncPod(event.ctx, event.pod)
+		log.G(ctx).WithField("podSyncronizer", id)
```

Comment:
Should it be used in the next line?

Reply:
ah darn... also should be event.ctx here.
Fixed.